[2024-11-02T21:30:18.570+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: table_dag.submit_tables scheduled__2024-10-24T00:00:00+00:00 [queued]>
[2024-11-02T21:30:18.608+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: table_dag.submit_tables scheduled__2024-10-24T00:00:00+00:00 [queued]>
[2024-11-02T21:30:18.616+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2024-11-02T21:30:18.747+0000] {taskinstance.py:1380} INFO - Executing <Task(PinotTableSubmitOperator): submit_tables> on 2024-10-24 00:00:00+00:00
[2024-11-02T21:30:18.822+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'table_dag', 'submit_tables', 'scheduled__2024-10-24T00:00:00+00:00', '--job-id', '264', '--raw', '--subdir', 'DAGS_FOLDER/table_dag.py', '--cfg-path', '/tmp/tmp_0hqnau6']
[2024-11-02T21:30:18.817+0000] {standard_task_runner.py:57} INFO - Started process 12847 to run task
[2024-11-02T21:30:18.834+0000] {standard_task_runner.py:85} INFO - Job 264: Subtask submit_tables
[2024-11-02T21:30:19.364+0000] {task_command.py:415} INFO - Running <TaskInstance: table_dag.submit_tables scheduled__2024-10-24T00:00:00+00:00 [running]> on host f79a3875bd34
[2024-11-02T21:30:20.392+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ctwang' AIRFLOW_CTX_DAG_ID='table_dag' AIRFLOW_CTX_TASK_ID='submit_tables' AIRFLOW_CTX_EXECUTION_DATE='2024-10-24T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-24T00:00:00+00:00'
[2024-11-02T21:30:20.476+0000] {pinot_table_operator.py:31} ERROR - Failed to submit table: 409 - {"code":409,"error":"Table config for customer_dim_OFFLINE already exists. If this is unexpected, try deleting the table to remove all metadata associated with it."}
[2024-11-02T21:30:20.495+0000] {pinot_table_operator.py:35} ERROR - An error occurred: Table submission failed with status code 409
[2024-11-02T21:30:20.609+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=table_dag, task_id=submit_tables, execution_date=20241024T000000, start_date=20241102T213018, end_date=20241102T213020
[2024-11-02T21:30:20.797+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-11-02T21:30:21.249+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
